File: Python/temperature.py
#This program converts a given temperature between Celsius and Fahrenheit.

print("---Temperature Convertor---")
print("1. C->F")
print("2. F->C")
n = int(input("Enter your choice : "))

if(n==1):
    temp = float(input("Enter the temperature : "))
    temp = temp*(9/5)+32
    print(f"F={temp:.2f}")

elif(n==2):         
    temp = float(input("Enter the temperature : "))
    temp = (temp-32)*5/9
    print(f"C={temp:.2f}")

else:        
    print("Invalid Choice!")

File: Java/temperature.java
// This program converts a given temperature between Celsius and Fahrenheit.

package Java_Samples;
import java.util.Scanner;

public class temperature {
    public static void main(String []agrs){
        Scanner sc = new Scanner(System.in);
        System.out.println("---Temperature Convertor---");
        System.out.println("1. C->F");
        System.out.println("2. F->C");
        System.out.print("Enter your choice : ");
        int n = sc.nextInt();
        
        switch(n){
            case 1 : System.out.println("Enter the temperature : ");
                     double temp = sc.nextDouble();
                     temp = temp*(9.0/5.0)+32;
                     System.out.printf("F=%.2f\n",temp);
                     break;

            case 2 : System.out.println("Enter the temperature : ");
                     temp = sc.nextDouble();
                     temp = (temp-32)*5.0/9.0;
                     System.out.printf("C=%.2f\n",temp);
                     break;

            default : System.out.println("Invalid Choice!");
        }

        sc.close();
    }
    
}
